apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -206,36 +214,45 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
           <p data-h2-margin-bottom="base(x0.5)">
             {intl.formatMessage({
               defaultMessage:
-                "You’ve now landed on your Digital Talent profile page.",
-              id: "ueLK1P",
+                "You've now landed on your Digital Talent profile page.",
+              id: "0Uq3M8",
               description:
                 "First paragraph for applicant dashboard notification welcoming an IAP user",
             })}
           </p>
           <p data-h2-margin-bottom="base(x0.5)">
             {intl.formatMessage({
               defaultMessage:
-                "The apprenticeship program is a part of a larger digital talent initiative in the Government of Canada that aspires to recruit awesome digital talent like yourself. This profile provides access to other digital opportunities and recruitments beyond the apprenticeship program, if you choose to apply. If you’d prefer to stick to the apprenticeship program exclusively, that’s okay too!",
-              id: "+im7hW",
+                "<strong>When you log into your account, you’ll start on this page from now on.</strong>",
+              id: "XOTFae",
               description:
-                "Second paragraph for applicant dashboard notification welcoming an IAP user",
+                "Notice that the application dashboard is the first page after logging in",
             })}
           </p>
-          <p>
+          <p data-h2-margin-bottom="base(x0.5)">
             {intl.formatMessage(
               {
                 defaultMessage:
-                  "When you log into your account, you’ll start on this page from now on. You can use the <strong><a>Track your applications</a></strong> section on this page to review your Indigenous apprenticeship application and track your status in the program.",
-                id: "RzNVWf",
+                  'The apprenticeship program is a part of a larger digital talent initiative in the Government of Canada that aspires to recruit top digital talent like yourself. You can use the "<a>Track your applications</a>" section on this page to review the status of your application to the IT Apprenticeship Program for Indigenous Peoples.',
```

Comment:
Just a reminder I think the `" "` in the translations will need that special character `« »`

Reply:
Good reminder!  I'll resolve this component when the translation comes back and I've confirmed.

Reply:
Yep, I had to manually fix that.  :wink: